source/_integrations/awair.markdown

```diff
@@ -10,56 +10,27 @@ ha_codeowners:
 ha_domain: awair
 ---
 
-The `awair` sensor platform will fetch data from your [Awair device(s)](https://getawair.com).
+The Awair integration will fetch data from your [Awair devices](https://getawair.com).
 
-You will need to request access to the Awair API and obtain an access token from the Awair [Developer Console](https://developer.getawair.com/).
+You will need to request access to the Awair API and obtain an access token from the Awair [Developer Console](https://developer.getawair.com/). It is free, but getting a token can take up to 24 hours.
 
 ## Configuring the Platform
 
-To enable these sensors, add the following lines to your `configuration.yaml` file:
-
-```yaml
-sensor:
-  - platform: awair
-    access_token: ACCESS_TOKEN
-```
-
-The Awair API has stringent usage quotas. The API method to discover devices in your account is
-limited to only 6 calls per 24 hours. If you find that you've exceeded this quota, you may optionally
-append device information to your configuration to bypass this call:
-
-```yaml
-sensor:
-  - platform: awair
-    access_token: ACCESS_TOKEN
-    devices:
-      - uuid: UUID
-```
-
-{% configuration %}
-access_token:
-  description: The access token for the Awair API.
-  required: true
-  type: string
-devices:
-  description: An optional list to manually configure devices rather than relying upon API discovery.
-  required: false
-  type: list
-  keys:
-    uuid:
-      description: UUID of the Awair sensor to monitor.
-      required: true
-      type: string
-{% endconfiguration %}
+To enable the Awair integration, add your Awair account from the integrations page.
 
 ## Available Sensors
 
-The platform will fetch all available sensors from each Awair device linked to your account. Supported sensors:
+The integration will fetch data from each device linked to your Awair developer account. The following sensors are supported:
 
   * Temperature
   * Humidity
-  * CO2
-  * VOC
-  * Dust, PM2.5, PM10: varies according to Awair model
+  * Carbon dioxide
+  * Total volatile organic compounds
+  * PM2.5 density
+  * PM10 density
+  * Sound level
+  * Luminescence
+
+Not all devices support all sensors; consult Awair's documentation to find out what sensors are present on your device. For first-generation Awair devices with a "dust" sensor, the integration will create identical PM2.5 and PM10 sensors (which reflects the capabilities of the sensor - it can detect dust between PM2.5 and PM10 but cannot differentiate between them).
 
-This platform refreshes at an interval based on a 300 API call per-day quota, and the number of devices you have configured.
```

Comment:
There is no quota anymore?

Reply:
Good catch. There are still quotas, but they are different now: the device list endpoint (used to discover the available sensors to add) is now unlimited, and fetching the latest data is now 300 per-device, rather than in total.

I've re-added a note about the default 300 per-device resulting in a refresh rate of once per five minutes. :smile:

Reply:
@ahayworth I checked the link to the documentation and it is not entirely clear to me if it indeed changed to 300 calls per device per day or not. It says per endpoint per day, but it is unclear to me if they count a different device as an endpoint or rather a different type of endpoint (like latest, 5min, ...). Have you tested this?

Reply:
Based on my testing, yes: it does seem like the rate limits are per-device, per day. But I agree that the docs are unclear. With that in mind: paging @deanlyoung ! Can you confirm or deny that the rate-limits for things like `air_data_latest` are indeed per-device, per-day?

Reply:
Confirming that the limits are per endpoint per device per 24 hours using UTC.

Reply:
Thanks!